sdk/greengrass/greengrass-client/src/event-stream-rpc-java/model/software/amazon/awssdk/aws/greengrass/model/SubscribeToTopicResponse.java

```diff
@@ -22,27 +23,34 @@ public boolean isVoid() {
     };
   }
 
+  /**
+   * @deprecated No longer used
+   */
   @Expose(
       serialize = true,
       deserialize = true
   )
+  @Deprecated
   private Optional<String> topicName;
 
   public SubscribeToTopicResponse() {
     this.topicName = Optional.empty();
   }
 
+  @Deprecated
   public String getTopicName() {
     if (topicName.isPresent()) {
       return topicName.get();
     }
     return null;
   }
 
+  @Deprecated
```

Comment:
Is this supposed to be deprecated? I didn't noticed deprecation warnings on the other PRs (though maybe I missed it?)